WooCommerce/WooCommerceTests/Tools/MockShippingLabelAddress.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Yosemite
+
+/// Generates mock `ShippingLabelAddress`
+///
+public struct MockShippingLabelAddress {
+    public static func sampleAddress(company: String = "",
```

Comment:
ℹ️  Perhaps this can be `empty()` too and we'll just use `copy()`.  🙂

Reply:
That'd be nice, though I haven't implemented `GeneratedCopiable` for `ShippingLabelAddress` yet - will do that the next time I update the tests since this PR is already big!